theano/tensor/basic.py

```diff
@@ -4573,7 +4573,7 @@ def reshape(x, newshape, ndim=None, name=None):
                 "to know what the number of dimensions of the reshaped "
                 "variable will be. You can provide the 'ndim' keyword "
                 "argument to 'reshape' to avoid this problem." % newshape)
-    op = Reshape(ndim, name)
+    op = Reshape(ndim)
```

Comment:
no, but a GPU must be available.

If you want to force a specific GPU, use this flag

init_gpu_device=gpu1


On Mon, Jul 25, 2016 at 11:09 AM, nke001 <notifications@github.com> wrote:

> In theano/tensor/basic.py
> <https://github.com/Theano/Theano/pull/4777#discussion_r72081535>:
>
> > @@ -4573,7 +4573,7 @@ def reshape(x, newshape, ndim=None, name=None):
> >                  "to know what the number of dimensions of the reshaped "
> >                  "variable will be. You can provide the 'ndim' keyword "
> >                  "argument to 'reshape' to avoid this problem." % newshape)
> > -    op = Reshape(ndim, name)
> > +    op = Reshape(ndim)
>
> @nouiz <https://github.com/nouiz> Do I have to specify anything in the
> command for the test to be run on GPU? thanks.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Theano/Theano/pull/4777/files/8ddd4ad580f78a9995f361b405934ba4d6adb2a7#r72081535>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AALC-04uzyQkt4eAlXoIUxoB5B9fBMJYks5qZNG9gaJpZM4JTuNL>
> .
>